django/db/backends/mysql/features.py

```diff
@@ -46,6 +46,7 @@ class DatabaseFeatures(BaseDatabaseFeatures):
             SET V_I = P_I;
         END;
     """
+    supported_explain_formats = {'JSON', 'TRADITIONAL'}
```

Comment:
I'd change `TRADITIONAL` to `TEXT` here as we are now implicitly converting `TEXT` to `TRADITIONAL` in your latest change to `explain_query_prefix()` for MySQL.

`TRADITIONAL` is just a formatted text-based output in MySQL, and a silly name.  I think it is better to provide a more sensible and consistent name for Django to use.

Reply:
Yep, good idea